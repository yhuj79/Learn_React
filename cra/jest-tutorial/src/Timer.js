import React from "react";

function Timer() {
  const time = Date.now();
  const hour = new Date(time).getHours();
  const min = new Date(time).getMinutes();
  const sec = new Date(time).getSeconds();

  return (
    <h1>
      {hour}시 {min}분 {sec}초
    </h1>
  );
}

export default Timer;
