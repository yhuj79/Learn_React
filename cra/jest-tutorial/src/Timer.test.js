import Timer from './Timer';

// Timer Test
test("Timer Test", () => {
  const el = <Timer />;
  expect(el).toMatchSnapshot();
});