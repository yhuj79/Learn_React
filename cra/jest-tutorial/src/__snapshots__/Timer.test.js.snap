// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer Test 1`] = `<Timer />`;
